tests/wallet/nft_wallet/test_nft_1_offers.py

```diff
@@ -1390,9 +1400,9 @@ async def test_complex_nft_offer(
     target_puzhash_taker = ph_taker
     royalty_puzhash_maker = ph_maker
     royalty_puzhash_taker = ph_taker
-    royalty_basis_pts_maker = uint16(200)
-    royalty_basis_pts_taker_1 = uint16(500)
-    royalty_basis_pts_taker_2 = uint16(500)
+    royalty_basis_pts_maker = uint16(0 if zero_royalties else 200)
+    royalty_basis_pts_taker_1 = uint16(0 if zero_royalties else 500)
+    royalty_basis_pts_taker_2 = uint16(0 if zero_royalties else 500)
```

Comment:
it might make sense to make these numbers separate test parameters. that way you can easily add more combinations to test